packages/locale/test/typescript_definitions/tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+	"compilerOptions": {
+		"target": "ES6",
+		"lib": ["ES2015"],
+		"module": "CommonJS",
+		"pretty": true,
```

Comment:
If pretty is true here, do we need the `--pretty` in the command in package.json?

Reply:
Not at all! I just missed it. I'll fix ASAP.